File: testprojects/java/benchmark/infrastructure/socialmedia/facebook/dao/MusicDAO.java
package infrastructure.socialmedia.facebook.dao;

import domain.facebook.Song;

//Functional requirement 3.1.2
//Test case 74: Only class presentation.gui.facebook.MusicGUI may have a dependency with domain.facebook.Song
//Result: FALSE
public class MusicDAO{
	//FR5.5	
	private Song song;

	public MusicDAO(){
		//FR5.2
		System.out.println(song.artist);
	}
}

File: testprojects/csharp/benchmark/infrastructure/socialmedia/facebook/dao/MusicDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.domain.facebook;

namespace CSharpBenchmark.infrastructure.socialmedia.facebook.dao
{
    //Functional requirement 3.1.2
    //Test case 74: Only class presentation.gui.facebook.MusicGUI may have a dependency with domain.facebook.Song
    //Result: FALSE
    public class MusicDAO
    {
        //FR5.5	
        private Song song;

        public MusicDAO()
        {
            //FR5.2
            Console.WriteLine(song.artist);
        }
    }
}